src/main/resources/application.properties.EXAMPLE

```diff
@@ -28,4 +35,3 @@ metadb.publishing_failures_filepath=
 
 # detailed consistency checking logger filepath
 metadb_req_filter.request_logger_filepath=
```

Comment:
[requested change] can change this to `smile_req_filter` or just not have metadb or smile in it at all and just make it  `request_filter.logger_filepath`? 

***** adding a reminder here to make these updates in the configuration repo as well